IPs/RSKIP125.md

```diff
@@ -0,0 +1,106 @@
+# Create2
+
+|RSKIP          |125           |
+| :------------ |:-------------|
+|**Title**      |Create2 |
+|**Created**    |29-APR-19 |
+|**Author**     | SMS |
+|**Purpose**    |Sca, Usa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+The purpose of this IP is to implement the CREATE2 opcode, as added to the Ethereum VM in the [EIP 1014](http://eips.ethereum.org/EIPS/eip-1014)
+
+## Motivation
+
+Allows interactions to (actually or counterfactually in channels) be made with addresses that do not exist yet on-chain but can be relied on to only possibly eventually contain code that has been created by a particular piece of init code. Important for state-channel use cases that involve counterfactual interactions with contracts.
```

Comment:
"Allow interactions" statement is confusing. Consider the shorter "Enables the creation of contracts with deterministic and repeatable addresses. This functionality is required for state channels having counterfactual interactions with a contract."

Reply:
This is the original motivation text from the EIP, I agree with you, it is confusing, I can make the change if you want anyways